020-Getting-started/020-installation.mdx

```diff
@@ -91,6 +91,15 @@ client = XataClient()
 ```
 </TabbedCode>
 
+### Committing files
```

Comment:
Sorry, my comments seem nit picky, but hoping to get our docs to align more with a style guide.  I think maybe using the Microsoft Style Guide as a ref going forward (because the [resources](https://github.com/MicrosoftDocs/microsoft-style-guide/tree/main) are open source), might be helpful. 

I think starting with a gerund for the heading is helpful for users as well as using more succinct sentences. Taking from your example and Dave's, I'd suggest the following reword (just a minor suggestion to what's already there... please let me know if I am missing some of the key points):
---------------------------------------

### Committing files to your repository

For collaborative development it's important to set up your local environment so that it can be replicated by others. For a team project, commit these specific files to your project's repository:

- `.xatarrc` - contains configuration information for your database setup. 
-  `src/xata.ts` - includes code related to the Xata client and API integration. This file shares the code that interacts with the Xata service.
- `.xata` - includes the migration files. This file manages changes to the database schema over time.

Committing these files to your project's  repo, enables team members to replicate your local setup by adding their own `XATA_API_KEY` to their `.env` file.

When creating templates or examples for public use and cloning, avoid committing files that contain specific database configurations. Some configuration files may not be suitable for others and can potentially expose personal information, posing security risks.

Instead, it's recommended to include the database schema in the repository using the command `xata init --schema schema.json`. The command initializes a database, using the schema information stored in the `schema.json` file, without sharing configuration files that might contain sensitive information.

You can then provide instructions on how to recreate the database in your project's README. You can also import CSV files to preload and existing schemas. Refer to the docs on [importing CSV data] (https://chat.openai.com/docs/csv-data/import-data-cli) for more information.

Reply:
Would be happy to also sync on this, because I am not very clear on the `xata init --schema ` part and would like to clarify.